core/src/main/resources/jenkins/management/Messages_tr.properties

```diff
@@ -23,32 +23,44 @@
 #
 
 ConfigureTools.DisplayName=Araçlar
-ConfigureTools.Description=Araçları, konumlarını ve otomatik kurucuları ayarla.
+ConfigureTools.Description=Araçları, konumlarını ve otomatik yükleyicileri yapılandırın.
```

Comment:
The Turkish word 'yapılandırma' is mainly reserved as the translation for the English word 'build' in Jenkins. As a result we use 'konfigürasyon' and 'ayar' for the translation of the English word 'configuration'. In your MR you are mixing these word with each other. This will cause confusion to the users.